docs/victory-axis/docs.md

```diff
@@ -22,6 +22,15 @@ The `tickCount` prop specifies approximately how many ticks should be drawn on t
 
 *default:* `tickCount={5}`
 
+### fixLabelOverlap
+
+When true, axis tick labels and their associated ticks
```

Comment:
This is a pretty technical/complicated explanation. Maybe we could open with something simple, saying this prop aims to only show a number of labels that will fit on the axis.

The note at the end about it only working with evenly-spaced labels is really helpful. In fact, it almost obviates the detailed explanation.